tests/unit/test_charm_TLS_enabled.py

```diff
@@ -52,7 +52,7 @@ def test_config_changed(self, mock_write_file, mock_check_call):
 
         mock_write_file.assert_any_call(
             "/usr/local/share/ca-certificates/nginx-server.crt",
-            "dGVzdF9jYV9jZXJ0==",
+            b64decode("dGVzdF9jYV9jZXJ0=="),
```

Comment:
To be more explicit and readable, better to use final const values where possible:

```suggestion
            b'test_ca_cert',
```

Same for the other instances of `b64decode('const string')` here.

Reply:
I tried using b'test_ca_cert' but it returned a padding error. Perhaps I should keep "dGVzdF9jYV9jZXJ0==" and make const?

Reply:
No, I mean to remove the `b64decode()` call altogether; see my suggested change above.

Reply:
Yeah I had some misunderstand there, just updated it, thanks.